starport_template/lib/stores/accounts_store.dart

```diff
@@ -202,6 +207,7 @@ class AccountsStore {
     return result.fold(
       (fail) {
         logError(fail);
+        errorDetails = fail.toString();
```

Comment:
we're loosing a lot of info here by calling `toString()`, couldn't we simply make errorDetails a, lets say `ErrorDetails` type that consists of the `dynamic error` and optional `StackTrace? stackTrace`.

Reply:
Will update this to use ErrorDetails

Reply:
created CosmosErrorDetails